modules/helfi_paragraphs_list_of_links/src/Entity/ListOfLinksItem.php

```diff
@@ -19,7 +22,7 @@ class ListOfLinksItem extends Paragraph implements ParagraphInterface {
   public function getDesign(): string {
     return $this->getParentEntity()
```

Comment:
`getParentEntity()` does not work correctly in `preSave()`. The parent is loaded from the database so the entity does not have the new value if it is edited.

See: https://git.drupalcode.org/project/paragraphs/-/blob/8.x-1.x/src/Entity/Paragraph.php?ref_type=heads#L128